custom_news_feed/custom_news_feed.py

```diff
@@ -225,10 +225,15 @@ def run(self):
 
     def get_news(self):
         """Get news content from JSON-file and display it."""
-        news_json_file_path = self.settings.value("CustomNewsFeed/json_file_path", None)
+        if self.settings.contains('/pythonplugins/customnewsfeedpath'):
```

Comment:
Siehe auch anderer PR, das musst du mir glaube ich nochmal erklären was hier genau passiert ;)

Reply:
In diesem Verzeichnis persistiere ich die Hash-Files zu jeder als gelesen markierten Meldung, damit ich weiss, welche der Anwender nicht mehr sehen will.